libs/extensions/std/lang/src/archive-interpreter-meta-inf.ts

```diff
@@ -19,7 +19,7 @@ export class ArchiveInterpreterMetaInformation extends BlockMetaInformation {
         archiveType: {
           type: PrimitiveValuetypes.Text,
           docs: {
-            description: 'The archive type to be interpreted, e.g., `"zip"`.',
+            description: 'The archive type to be interpreted, e.g., `"zip" or "gz`.',
```

Comment:
```suggestion
            description: 'The archive type to be interpreted, e.g., `"zip" or "gz".',
```

Reply:
Changed to 'The archive type to be interpreted, e.g., "zip" or "gz".',